docs/userguide/containers/dockervolumes.md

```diff
@@ -135,8 +143,8 @@ Docker `-v` flag.
 Mounting a host directory can be useful for testing. For example, you can mount
 source code inside a container. Then, change the source code and see its effect
 on the application in real time. The directory on the host must be specified as
-an absolute path and if the directory doesn't exist Docker will automatically
-create it for you.  This auto-creation of the host path has been [*deprecated*](#auto-creating-missing-host-paths-for-bind-mounts).
+an absolute path and if the directory doesn't exist the Engine daemon automatically
+creates it for you.  This auto-creation of the host path has been [*deprecated*](#auto-creating-missing-host-paths-for-bind-mounts).
```

Comment:
This will most likely be removed in 1.11 through https://github.com/docker/docker/pull/19953. But we can remove in that PR